### DISCUSSION TITLE
Interaction.edit

### DISCUSSION THREAD

**Discussion by malikmajai** [mccoderpy/discord.py-message-components#20] [General]
Why I can't edit an interaction twice?
![image](https://user-images.githubusercontent.com/92770716/166161616-78622bd6-7cc9-49dd-b462-7973e9b75c50.png)

**Comment by mccoderpy** [comment]
Should bee possible. Do you get any erros? But you should be able to edit it multiple times by putting the first edit in a variable like 
```py
mes = await interaction.edit(...)

await mes.edit(...)
```

**Reply by malikmajai** [reply]
Let me try.

**Reply by malikmajai** [reply]
![image](https://user-images.githubusercontent.com/92770716/166162107-3625ac59-3b2d-4da7-a040-71d7aedab7e2.png)

**Reply by mccoderpy** [reply]
Hm.. lol ... try use
```py
await interaction.callback_message.edit(...)
```
Otherwise I can't fix it right now because I have a 3 hour drive ahead of me

**Reply by malikmajai** [reply]
Same error as above. Have a Safe Journey.

**Reply by mccoderpy** [reply]
Thank you and sorry for the problems. You vould try to create a PartialMessage object with the message id and edit this one.

**Reply by malikmajai** [reply]
```interaction.message.edit``` works. But I have main embed interface there. I just wondered if it is possible to edit interaction multiple times. Like after a ```await interaction.respond(...)```

**Reply by mccoderpy** [reply]
The Interaction.edit is a bit weird atm. 
But yes, in general you can.
